src/__tests__/Integration.test.js

```diff
@@ -0,0 +1,24 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { fetchCryptoData } from '../redux/crypto';
+import cryptoData from '../mock/cryptoData';
+
```

Comment:
![image](https://user-images.githubusercontent.com/43698511/142736714-0bd7879d-c42a-46b4-a686-13b611443805.png)

- [ ] Kindly add some integration tests using React testing library. You can check [this tutorial](https://www.robinwieruch.de/react-testing-library/) for tips.